libdnf/rpm/transaction.hpp

```diff
@@ -334,6 +334,7 @@ class Transaction {
 
     std::map<unsigned int, rpmte> implicit_ts_elements;  // elements added to the librpm transaction by librpm itself
     bool downgrade_requested{false};
+    bool ignorearch{false};
```

Comment:
Do we need a private copy of `ignorearch` stored in the class?

Reply:
I think that the value could be used directly from `base.conf`. Or is there any reason why would you like to store it here?

Reply:
No reason, I've changed it to use the config option directly.